src/services/auth/login.ts

```diff
@@ -0,0 +1,35 @@
+import axios from 'axios';
+// Own
+import { API_BASE_URL } from 'config/constants';
+import BackendError from 'exceptions/backend-error';
+
+const URL = `${API_BASE_URL}/login`;
+
+export default async function login(body: LoginBody): Promise<LoginResponse> {
+  try {
+    //const response = await axios.post<LoginResponse>(URL, body);
+    //console.log('lo chido', response);
+    return new Promise((resolve, reject) => {
+      resolve({
+        name: 'Hector',
+        email: 'email@gmail.com',
+        token: 'token'
+      });
+    });
```

Comment:
```suggestion
    return new Promise((resolve, reject) => {
      resolve({
        name: 'Hector',
        email: 'email@gmail.com',
        token: 'token'
      });
    });
```

papapapa dejalo para que furule con el backend de una